easybuild/easyconfigs/p/protobuf/linking-error.patch

```diff
@@ -0,0 +1,36 @@
+diff -ruN protobuf-21.9-orig/src/libprotobuf-lite.map protobuf-21.9/src/libprotobuf-lite.map
```

Comment:
@guacke please add short description and author info (or add link to https://github.com/protocolbuffers/protobuf/pull/11032) at the top of the patch file (and update the checksum).

please also rename the patch as `protobuf-21.9_linking-error.patch` (make sure to remove the old one)

Reply:
+1, please include a reference to https://github.com/protocolbuffers/protobuf/pull/11032